### ISSUE TITLE
Can't disable site specific holiday snow in Calypso for simple site (wp-admin gap)

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-calypso#20395]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/settings/general/testsite.wordpress.com

#### What I expected

To see site specific holiday snow option as per wp-admin, or for a Jetpack site in Calypsp

#### What happened instead

No setting

#### Browser / OS version

Safari, Chrome, Firefox on macOS

#### Screenshot / Video

Jetpack Site:

<img width="743" alt="screen shot 2017-12-01 at 2 52 10 pm" src="https://user-images.githubusercontent.com/128826/33468343-45a40a72-d6a7-11e7-9a99-c5789ae31d6b.png">

Simple Site:

<img width="753" alt="screen shot 2017-12-01 at 2 52 56 pm" src="https://user-images.githubusercontent.com/128826/33468351-5c460b18-d6a7-11e7-9a17-b47072e75d8f.png">

#### Context / Source
#dogfooding, #manual-testing,

**Comment by KokkieH** [comment]
Already had one request on how to disable it in 884787-f. Pointed them to WP-Admin for now, but adding here for follow-up once the above PR is merged.

**Comment by alisterscott** [comment]
There's a pending fix in #20397

**Comment by jobthomas** [comment]
@alisterscott happy to add a separate issue for this (and is this the right repo for that?)

The default enabling of snow seems very Northern Hemisphere-centric - would it be possible to only enable by default if the user is registered in the Northern Hemisphere.

**Comment by alisterscott** [comment]
@jobthomas yes, if you could raise this as an enhancement in this repository that would be great. There are two snow settings - one is for a customer's own site or sites - whether other visitors see snow on those sites, and the other is whether an individual customer sees snow on other WordPress.com sites when logged in - please specify which one or whether both should default based on geolocation (hemisphere)
